### PR TITLE
Add authticket table migration

### PR DESCRIPTION
**Pull request by chdorner**
This is the migration for the table that will be used as the storage for the authentication tickets coming up in the next pull request.

### CODE REVIEW

File: h/migrations/versions/1e88c31d8b1a_add_authticket_table.py [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+                    sa.Column('expires',
+                              sa.DateTime,
+                              nullable=False),
+                    sa.Column('user_id',
+                              sa.Integer(),
+                              nullable=False),
+                    sa.Column('user_userid',
+                              sa.UnicodeText(),
+                              nullable=False),
+                    sa.ForeignKeyConstraint(['user_id'], ['user.id']))
```
Comment (nickstenning) [inline h/migrations/versions/1e88c31d8b1a_add_authticket_table.py]: Shall we add on `ondelete='cascade'` here?

Comment (chdorner) [inline h/migrations/versions/1e88c31d8b1a_add_authticket_table.py]: Yes! Forgot about that

Comment (chdorner) [inline h/migrations/versions/1e88c31d8b1a_add_authticket_table.py]: Done

**Comment by nickstenning** [comment]
LGTM.

**Pull request merged**